utils/docker-utils.sh

```diff
@@ -145,25 +212,32 @@ docker::deploy_sanity() {
 
     log info "Checking for deployment errors..."
     for i in "$@"; do
+        log info "Waiting for $i to start ..."
         local start_time
         start_time=$(date +%s)
 
+        error_message=()
         until [[ $(docker service ps instant_"$i" --format "{{.CurrentState}}" 2>/dev/null) == *"Running"* ]]; do
-            config::timeout_check "${start_time}" "$i to start"
+            config::timeout_check "${start_time}" "$i to start" 60 30
```

Comment:
It would be nice to make this more robust against the downloading of images. This time limit is nice if the images already exist, but the time limit is too short if the images don't exist locally.

Reply:
Yeah good point of vue ! fixed in [fe3ef41](https://github.com/jembi/platform/pull/188/commits/fe3ef4174075cafb347d1e48d0517b936e93f604)